pkg/client/cli/cmd_status.go

```diff
@@ -236,6 +241,8 @@ func (cs *userDaemonStatus) WriteTo(out io.Writer) (int64, error) {
 		n += ioutil.Printf(out, "  Executable        : %s\n", cs.Executable)
 		n += ioutil.Printf(out, "  Install ID        : %s\n", cs.InstallID)
 		n += ioutil.Printf(out, "  Status            : %s\n", cs.Status)
+		n += ioutil.Printf(out, "    Mode            : %s\n", cs.Mode)
+		n += ioutil.Printf(out, "    Client Count    : %d\n", cs.ClientCount)
```

Comment:
These are not user daemon statuses. They are traffic manager statuses. If we want to print status info about the traffic-manager, then should put it under a new "Traffic Manager:" heading (similar to the version command).